tests/fixtures.py

```diff
@@ -34,7 +34,7 @@ def setup_index_tests_integration(setup_test_env):
 
     with TemporaryDirectory() as tempdir:
         deployer = GithubPagesDeployer(Path("doc/"), source_branch, "origin", current_commit_id.stdout[:-1], ["../test_package"],
-                                       target_branch, "origin", "push",
+                                       target_branch, "origin", True,
```

Comment:
If you have a boolean type in an API, keyword parameter are preferred, because then the reader knows which parameter it is. For this particular call, I would use keyword parameter for all parameter to make it easier to understand.

Reply:
done